build_deploy.sh

```diff
@@ -1,4 +0,0 @@
-#!/bin/bash
-# This is a placeholder file needed by the CI/CD templates
-# You can safely ignore this file :)
-exit 0
```

Comment:
are you sure this should be removed? last time I talked to @aditya-konarde he mentioned that this is needed by the CI.

Reply:
grrr this is true